res/database/Default/user/scr0llbaer_screenplays_adaptations.xml

```diff
@@ -364,10 +367,12 @@
 				</scripttemplate>
 				<scripttemplate index="2" product="2" licence_type="2" guid="f064c96f-0fbf-4187-93ee-2313f545c259">
 					<title>
-						<all>Hod</all>
+						<de>Hod</de>
```

Comment:
Mainly because Polish entries for these will be different.

Also for "real" titles when two languages match more or less only by coincidence (and also some other episode titles don't, like in this instance) it "feels" a bit wrong to use `<all>` (not that I have followed this rule consistently...)